fahrschein/src/test/java/org/zalando/fahrschein/NakadiClientTest.java

```diff
@@ -302,11 +303,11 @@ public void shouldPublishEventsWithRequestHandler() throws IOException {
 
 
     @Test
-    public void shouldPublishEventsWithRequestHandlerAndFail() throws IOException {
+    public void shouldPublishEventsWithRequestHandlerAndFailWithUnprocessableEntity() throws IOException {
         server.expectRequestTo("http://example.com/event-types/foobar/events", "POST")
                 .andExpectJsonPath("$[0].id", equalTo("1"))
                 .andExpectJsonPath("$[1].id", equalTo("2"))
-                .andRespondWith(207, ContentType.APPLICATION_JSON, "[{\"eid\":\"event-one\",\"publishing_status\":\"failed\",\"step\":\"validating\",\"detail\":\"baz\"}]")
+                .andRespondWith(422, ContentType.APPLICATION_JSON, "[{\"eid\":\"event-one\",\"publishing_status\":\"aborted\",\"step\":\"validating\",\"detail\":\"baz\"}]")
```

Comment:
```suggestion
                .andRespondWith(422, ContentType.APPLICATION_JSON, "[{\"eid\":\"event-one\",\"publishing_status\":\"failed\",\"step\":\"validating\",\"detail\":\"baz\"}]")
```

only the following events are `aborted`